.circleci/config.yml

```diff
@@ -11,29 +16,46 @@ jobs: # a collection of steps
       - run:
           name: update-npm
           command: "sudo n stable"
-      - restore_cache: # special step to restore the dependency cache
-          # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
-          key: dependency-cache-{{ checksum "server/package.json" }}
+      # - restore_cache: # special step to restore the dependency cache
+      #     # Read about caching dependencies: https://circleci.com/docs/2.0/caching/
+      #     key: dependency-cache-{{ checksum "server/package.json" }}
       - run:
           name: install-npm-wee
           command: cd server && npm install
-      - save_cache: # special step to save the dependency cache
-          key: dependency-cache-{{ checksum "server/package.json" }}
-          paths:
-            - server/node_modules
+      # - save_cache: # special step to save the dependency cache
+      #     key: dependency-cache-{{ checksum "server/package.json" }}
+      #     paths:
+      #       - server/node_modules
```

Comment:
can we remove these commented lines of code if we are not using them

Reply:
I commented them when trying to fix the ci.
Uncommented as still relevant to use cache.